### PR TITLE
feat: rules page added

### PR DESCRIPTION
**Pull request by Developers-codz**
Done with rules page
rules

Thank you for reviewing my PR :)

### CODE REVIEW

File: styles/rules.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+.rules {
+  display: inline-block;
+}
+.btn-start {
+  text-decoration: none;
+  font-size: 1.3rem;
+  border: 2px solid var(--primary);
+  color: var(--primary);
+  padding: 0.5rem 1rem;
```
Comment (b0llu) [inline styles/rules.css:32]: you can use px if the value is less than 1rem

**Review by b0llu — APPROVED** [review]
Only one suggestion everything else looks good

File: styles/rules.css (hunk 1 above)
Comment (Developers-codz) [reply styles/rules.css:32]: I am comfortable with rem :)

File: index.html [hunk 2]
```diff
@@ -59,28 +59,28 @@ <h1 class="mb-lg header-logo font1">B-Quiz</h1>
       <h1 class="category" id="categories">Categories</h1>
       <div class="category-wrapper">
         
-          <a href="#" class="card-category shadow-box">
+          <a href="rules.html" class="card-category shadow-box">
             <img src="./assets/painting.jpg" alt="painting" class="category-image">
             <div class="category-content">
               <h3 class="font3 category-head">Paintings</h3>
               <h4 class="">1 Quiz</h4>
```
Comment (animeshgarai09) [inline index.html]: remove extra class attribute

File: index.html [hunk 3]
```diff
@@ -59,28 +59,28 @@ <h1 class="mb-lg header-logo font1">B-Quiz</h1>
...
             </div>
           </a>
-          <a href="#" class="card-category shadow-box">
+          <a href="rules.html" class="card-category shadow-box">
             <img src="./assets/painting.jpg" alt="painting" class="category-image">
             <div class="category-content">
               <h3 class="font3 category-head">Paintings</h3>
               <h4 class="">1 Quiz</h4>
```
Comment (animeshgarai09) [inline index.html]: same here

File: styles/rules.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+.fa-gavel {
+  color: var(--primary);
+  margin-right: 2rem;
+}
+.rules {
+  display: inline-block;
+}
+.btn-start {
+  text-decoration: none;
```
Comment (Maianki) [inline styles/rules.css:28]: Suggestion: Try giving space between two CSS rules. It improves readability of the code.

**Review by Maianki — APPROVED** [review]
Great work Kajal, Good use of semantic HTML. Just had one suggestion.

File: styles/rules.css (hunk 4 above)
Comment (Developers-codz) [reply styles/rules.css:28]: Prettier is looking into that :)

File: index.html (hunk 3 above)
Comment (Developers-codz) [reply index.html]: Done ! Thank you for pointing that out

**Review by animeshgarai09 — APPROVED** [review]
Looks good

**Pull request merged**